config/cypress/cypress.json

```diff
@@ -0,0 +1,11 @@
+{
+   "baseUrl": "http://localhost:8001",
+   "fixturesFolder": "../../tests/fixtures",
```

Comment:
Cypress fixtures can be disabled, as we are not using `cy.fixture`. See https://docs.cypress.io/api/commands/fixture.html#Syntax.

To disable, do: `"fixturesFolder": false`.